src/client/utils/styles/_css-variable.scss

```diff
@@ -0,0 +1,10 @@
+// sass-lint:disable no-duplicate-properties
+@mixin css-variable($prop, $var) {
+  #{$prop}: #{map-get($variables, $var)};
+  #{$prop}: var(--#{$var});
```

Comment:
Is it possible that the `$var` is not defined in the runtime, but it's present in `$variables` map, so using it as the default value (the second parameter of `var` function) makes sense?

Reply:
No, that is never the case. There is code to automatically defines a CSS var for every entry in `$variables`